plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -15,6 +15,20 @@
 // limitations under the License.
 //
 
+
+proc mayaUsdTranslatorImport_hideInfo()
+{
+    if (`rowLayout -exists mayaUsd_NBPrimsInScopeRL`) 
+    {
+        rowLayout -edit -visible false mayaUsd_NBPrimsInScopeRL;
+        rowLayout -edit -visible false mayaUsd_NBSwitchedVariantsRL;
+
+        text -edit -label "---" mayaUsd_NBPrimsInScopeLabel;
+        text -edit -label "---" mayaUsd_NBSwitchedVariantsLabel;
```

Comment:
These text labels are hidden if we don’t have current info to display, so technically we don’t have to come up with a display that indicates that the value hasn’t been set, but I do it anyways.